### ISSUE TITLE
Error in recognition of the .par file

### ISSUE THREAD

**Issue by uhaifabioinfo** [compomics/peptide-shaker#105]
Hi,

I have run a search in SearchGUI, followed by Peptideshaker. one of the modifications I've added is a novel one that appears in Unimod but not in Peptideshaker. at the end of the run PeptideShaker was aborted, claiming several other modifications have been altered. It's important to note that this occurred after updating to 1.3, although using 1.2 didn't yield any benefit.
It seems that none of the parameter files I'm preparing is saved and they are all blank, even when I try to save them or export to another file.

Any help would be greatly appreciated.

Cheers,
David.

**Comment by hbarsnes** [comment]
Hi David,

Thanks for telling us about this. Could you send us your PeptideShaker log file? You'll find it via the Welcome dialog > Settings & Help > Help > Bug Report.

I assume from the above that you're running PeptideShaker from SearchGUI, is that correct? If so, could you try loading the search results directly in PeptideShaker as well?

And regarding your parameter files being empty, is this the case just after they are saved, or only after trying to load the search results in PeptideShaker?

And finally, could you send us the details of the modification you are trying to add? We can then try to add the same modification on our side to try to reproduce the issue.

Best regards,
Harald

**Comment by uhaifabioinfo** [comment]
Hi Harald,

 

Please see the attached log for peptideshaker and searchGui.

 

I’ve tried to upload the results straight on peptideshaker and was notified that it couldn’t upload the parameters – both logs are attached. 

The parameter file appears to be empty anytime I try to upload to peptideshaker – either after the SearchGUI search or independently. The file doesn’t appear to be empty when it’s opened by Notepad++ (I’m attaching a parameter file), but when I open it using the EDIT function, there is nothing there – database, mods or mass tolerances. 

The modification I’m after it carboxylation of D and E. it was set to generate neutral losses of the carboxyl in HCD.

 

Thanks!

David.

**Comment by hbarsnes** [comment]
Hi David,

The issue seems to be related to this one: https://groups.google.com/forum/#!topic/peptide-shaker/TFWdjeYxfEs.

I am in the process of releasing new versions of both SearchGUI (v2.3.1) and PeptideShaker (v1.3.1).

Please give these a go when they become available and let me know if the fix solves your issue as well.

Best regards,
Harald

**Comment by uhaifabioinfo** [comment]
Thank you!

**Comment by uhaifabioinfo** [comment]
Hi Harald,

 

 

I’ve installed and re-installed both peptideshaker and searchGUI. Now the problem is that peptideshaker stalls when loading the data or crashes during the processing quite fast – see the attached log file.

Just to try and narrow the problem, I’ve tried and succeeded uploading an old results file (the last run on the log).

 

Thanks!

David.

**Comment by hbarsnes** [comment]
Hi David,

Did you recreate your parameter file and redo the search? As otherwise the previous issue will still result in issues with the parameter file when trying to open it in PeptideShaker.

It also seems like you're having a problem with your mgf file indexing. Could you try deleting the related .mgf.cui file and see if that helps? Have you successfully processed this mgf file before?

From which search engine are you trying to load data when the processing fails?

Best regards,
Harald

**Comment by uhaifabioinfo** [comment]
Hi Harald,

 

I erased all of the parameter files prior to the upgrade. But the .mgf may be a problem – I’ve received it from a collaborator and that may well be a problem.

I am going to test the software on another dataset I have and we’ll see.

 

Will keep you updated.

 

Thanks!

David.

**Comment by hbarsnes** [comment]
Hi David,

It would still be great to know which search engine results PeptideShaker is trying to load when getting these problems? Maybe it's a more general issue on our side. For example, we're seeing some problems with loading Comet results after removing our mgf to ms2 conversion and instead giving the mgf file directly to Comet.

Best regards,
Harald

**Comment by uhaifabioinfo** [comment]
I

Hi Harald,

 

Doesn’t the file says it? I don’t think it began to load results really, but I did have Comet and Tide (and MS-GF and MS-Amanda) searching. I might just run MS-GF+ and see.

 

Thanks!

David.

**Comment by uhaifabioinfo** [comment]
Hi Harald,

 

Tried a completely different dataset (straight from RAW files using proteowizard on searchGUI) – peptideshaker aborts with the following message:

The definition of the following PTM(s) seems to have changed and were overwritten:

Pyrolidone from Q, Oxidation of M, Amidation of the protein C-term and Carbamidomethylation of C.

Please verify the definition of the PTM(s) in the modifications editor.

 

See attached file..

 

Thanks!

David.

**Comment by hbarsnes** [comment]
Hi David,

You should be able to simply ignore the "PTM(s) seems to have changed" message, it should not cancel the processing.

However, I do see another error in your log file related to trying to figure out of a peptide is at the C-terminal end of a given protein. What kind of database are you searching against? Is it a peptide database? Or does it contain some very short protein sequences?

Best regards,
Harald

**Comment by hbarsnes** [comment]
Hi again,

Could you try this beta version to see if it allows you to load your data: https://www.dropbox.com/s/qnaqucwhle3zqqr/PeptideShaker-1.3.2-beta.zip?dl=0?

It should at least solve the issues in the methods for detecting if a peptide is a C- or N-terminal peptide in a given protein.

Best regards,
Harald

**Comment by uhaifabioinfo** [comment]
Hi Harald,

 

It’s an antibody DB from Abysis - it does contain peptide fragments. 

Question – can I use a peptidic DB generated by PEAKS (or Novor, etc.)?

 

Thanks for the fix – will keep you updated.

 

Cheers,

David.

**Comment by uhaifabioinfo** [comment]
Hi Harald,

 

The fix works, thanks!

 

David.

**Comment by hbarsnes** [comment]
Hi David,

Great! I'll release a new official release as soon as possible. Will let you know when it's online.

> Question – can I use a peptidic DB generated by PEAKS (or Novor, etc.)?

Yes, as long as the FASTA headers can be parsed correctly. See https://github.com/compomics/searchgui/wiki/DatabaseHelp for details. And if your FASTA file cannot be parsed, please let is know.

BTW, if you use Novor you might want to take a look at DeNovoGUI: http://compomics.github.io/projects/denovogui.html.

Best regards,
Harald

**Comment by hbarsnes** [comment]
Hi David,

I've now released the official version of PeptideShaker 1.3.2. Please update to this version instead of the beta version, as the beta version will not inform you about new releases.

I'm now closing this issue, but please open a new issue if you come across other problems in the future.

Best regards,
Harald

**Comment by uhaifabioinfo** [comment]
Thank you for the support!

 

David.
